### PR TITLE
Remove IFluidHTMLView from examples/data-objects/webflow

### LINKED ISSUE #9963: Remove IFluidHTMLViewFactory usage from @fluid-example/webflow
As a part of https://github.com/microsoft/FluidFramework/issues/1041 we are looking to deprecate view-related interfaces to better align with popular UI frameworks and patterns adopted there.

This works item tracks removal of `IFluidHTMLViewFactory` usage from @fluid-example/webflow.

### PR DESCRIPTION
**Pull request by ms-avocado**
PR for https://github.com/microsoft/FluidFramework/issues/9963

Removes all usages of IFluidHTMLView from examples/data-objects/webflow

### CODE REVIEW

File: examples/data-objects/webflow/src/host/host.ts [hunk 1]
```diff
@@ -13,7 +12,8 @@ import { debug } from "./debug";
 import * as styles from "./index.css";
 import { SearchMenuView } from "./searchmenu";
 
-export class WebflowView implements IFluidHTMLView {
+export class WebflowView {
```
Comment (ChumpChief) [inline examples/data-objects/webflow/src/host/host.ts:15]: You'll need to revert these changes and also in package.json for this PR -- converting `WebflowView` to no longer be an `IFluidHTMLView` can be its own PR (probably converting it to React).

File: examples/data-objects/webflow/src/html/formatters.ts [hunk 2]
```diff
@@ -70,7 +68,6 @@ class HtmlFormatter extends RootFormatter<IFormatterState> {
 interface IInclusionState {
     root?: HTMLElement;
     slot?: HTMLElement;
-    view?: Promise<IFluidHTMLView>;
 }
 
 export class InclusionFormatter extends Formatter<IInclusionState> {
```
Comment (ChumpChief) [inline examples/data-objects/webflow/src/html/formatters.ts:73]: You can probably go further and delete all the inclusion stuff entirely -- looks like no one is calling `FlowDocument.insertComponent()` which should mean that inclusions don't really exist currently.

File: examples/data-objects/webflow/src/view/layout.ts [hunk 3]
```diff
@@ -92,7 +91,7 @@ export class Layout extends EventEmitter {
     private renderPromise = done;
     private renderResolver: () => void;
 
-    constructor(public readonly doc: FlowDocument, public readonly root: Element, formatter: Readonly<RootFormatter<IFormatterState>>, public readonly viewFactoryRegistry: Map<string, IFluidHTMLViewFactory> = new Map(), public readonly scope?: FluidObject) {
+    constructor(public readonly doc: FlowDocument, public readonly root: Element, formatter: Readonly<RootFormatter<IFormatterState>>, public readonly scope?: FluidObject) {
```
Comment (ChumpChief) [inline examples/data-objects/webflow/src/view/layout.ts:94]: While you're here, could you also please remove `scope`?  Looks like it was only potentially used by the view factories.

File: examples/data-objects/webflow/src/host/host.ts (hunk 1 above)
Comment (ronnymajani-microsoft) [reply examples/data-objects/webflow/src/host/host.ts]: fixed in d9e3a6d27b08676bc8d282d4e082c45a4b31582c

File: examples/data-objects/webflow/src/html/formatters.ts (hunk 2 above)
Comment (ronnymajani-microsoft) [reply examples/data-objects/webflow/src/html/formatters.ts]: fixed in d9e3a6d27b08676bc8d282d4e082c45a4b31582c

File: examples/data-objects/webflow/src/view/layout.ts (hunk 3 above)
Comment (ronnymajani-microsoft) [reply examples/data-objects/webflow/src/view/layout.ts]: fixed in f9d718e3959316cd59736cf00d497f52330f54f8

**Review by ChumpChief — APPROVED** [review]
Looks good!  I put one more suggestion and I think this will be good for checkin.

We can probably go further on removing the inclusion infrastructure for this demo, but since it's not really necessary in order to get the `IFluidHTMLViewFactory` removed, seems fine to me to leave it for later.

File: examples/data-objects/webflow/src/document/index.ts [hunk 4]
```diff
@@ -317,7 +315,6 @@ export class FlowDocument extends LazyLoadedDataObject<ISharedDirectory, IFlowDo
     // eslint-disable-next-line @typescript-eslint/ban-types
     public insertComponent(position: number, handle: IFluidHandle, view: string, componentOptions: object, style?: string, classList?: string[]) {
```
[the remark this answers is not in the document]
Comment (ronnymajani-microsoft) [reply examples/data-objects/webflow/src/document/index.ts]: done afe727411109026942cfe4352b8ea7176993ddba

**Comment by ssimic2** [comment]
You took it a bit further then what ticket spec-ed out, nice work!

**Review by ssimic2 — APPROVED** [review]
